cosmos/settings.py

```diff
@@ -34,6 +34,10 @@
 remote_cache_dir = conf.get("cosmos", "remote_cache_dir", fallback=None)
 remote_cache_dir_conn_id = conf.get("cosmos", "remote_cache_dir_conn_id", fallback=None)
 
+dbt_compile_task_id = conf.get("cosmos", "dbt_compile_task_id", fallback="dbt_compile")
```

Comment:
Cosmos already has so many configurations; perhaps we should have this as a constant. I don't think we're giving much value for them to customise this name.

Reply:
removed this config and using it as a constant